### ISSUE TITLE
Highlightr No Longer Working w Obsidian v0.15.3

### ISSUE THREAD

**Issue by Paddy1058** [chetachiezikeuzor/Highlightr-Plugin#41]
I was enjoying using Highlightr, but it no longer works. Only thing I can identify is recent update to Ob v0.15.3. Have tried uninstalling and reinstalling plugin.

Anyone else having this problem?  Solutions?

**Comment by chetachiezikeuzor** [comment]
I apologize for the lack of updates to this plugin. I will take a look at this tonight and see what I can do. Thank you for your patience!

**Comment by Paddy1058** [comment]
FYI, I found a partial work a round -- by setting hotkeys for the various colors, I can now again add highlights. But setting a hot key to open the highlightr doesn't work -- it just doesn't respond, and doesn't display the choice of colors. Also, to unlighlight, right-clicking on the highlight doesn't bring up the unhighlight menu, but raises a note that "focus must be in edito" (I'm using live preview). Clicking on the highlight puts it in edit mode, then right-clicking brings up unhighlight option, and clicking then unhighlights. Hope this may help in troubleshooting.  Really like this plugin when it was working normally.

**Comment by Paddy1058** [comment]
I found another factor affecting this plugin: whether the Obsidian css is turned on or off in Settings/Appearance, down at the bottom of the page. If it is turned on, then the highlight appears but is not the chosen color. Turning the css off then allows the chose color highlight to appear -- although still very limited, as described above. Of course, this might depend on the css file in the vault.

**Comment by remerle** [comment]
I have a workaround, and possibly a starting point for @chetachiezikeuzor's investigation. It seems the Obsidian API expects you to play nice with their context menus, so I suspect a fix would require using the [`menu.addItem`](https://marcus.se.net/obsidian-plugin-docs/user-interface/context-menus) method rather than overwriting the dom.

You can find the crude workaround here: https://github.com/chetachiezikeuzor/Highlightr-Plugin/issues/42#issuecomment-1203250330

**Comment by Paddy1058** [comment]
Yup, fix works -- see https://github.com/chetachiezikeuzor/Highlightr-Plugin/issues/42#issuecomment-1211549177
